sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -178,7 +178,7 @@ export interface Pipeline {
 export interface PipelineOptions {
     proxyOptions?: ProxySettings;
     redirectOptions?: RedirectPolicyOptions;
-    retryOptions?: ExponentialRetryPolicyOptions;
+    retryOptions?: ExponentialRetryPolicyOptions | ThrottlingRetryPolicyOptions;
```

Comment:
My preference is to define a new interface that is morally the superset of all retry policies (in the near future this can be the options of our unified retry policy):

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgArAA4QDaggJQjCgE8B5DMYAexAGdkBvAKGTeQFs4APQ44CHQD8ALmQgArhwBG0ANyt2UIqQAiOOCQCSIALLCxkmfMVsuvFSXXZNO-aPFTZUBQF9mzUJFiIUAUW4MWghwYDhsPlIKKloGFnZOHkiBA0djF1NkZWIrDW09VKNnBQTzSNybfPtDJxN3T3BoeCRkABUACyhqMDBcEABzcuiaeiZMspUUhyK6j2YYCRAEGJBkCEDg0PDy1GpcBBIACmpKEboxAKCQEKpty2HYgEox+oWlleQwTu7e0EHLXb7I4nFbnNrfHp9f45B70Z4serYIjIEFnMToLB9Aj3U6xZAAXiYrgUzHWVxuYQiAL2wAOx1xcIUXy6kL+OxpdNRTzkQA

It's a small amount of duplication, but I think it'll pay off when the individual policies go away and it makes for nicer IntelliSense than "a | b | c" especially when those highly overlap

Reply:
Done!